app/views/welcome/index.html.erb

```diff
@@ -1,45 +1,55 @@
-    </p>
-    <p>
-      Our charity works to collect clothing, furniture, food, toys, books, hygiene products, and other miscellaneous materials for the homeless and people in need.
-    </p>
-    <p>
+<div class="index">
+  <div class="heart-box">
+    <h2>Welcome</h2>
+    <div class="row">
+      <div class="large-7 large-centered columns">
+        <p>
+          Our charity works to collect clothing, furniture, food, toys, books, hygiene products, and other miscellaneous materials for the homeless and people in need. Resources that may help low income families, and those after leaving shelters are especially appreciated.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div class="accept-box">
+    <h2>
       We accept all kinds of donations to help those in need!
-    </p>
-    <ul>
-      <li>
-        Donations of food, hygiene products, clothing, and furniture are especially appreciated by the less fortunate.
-      </li>
-      <li>
-        Resources that may help low income families, and those after leaving shelters.
-      </li>
-      <li>
-        We also welcome leads, and information to other organizations that can provide assistance.
-      </li>
+    </h2>
+
+    <ul class="small-block-grid-2 large-block-grid-4">
+      <li><%= image_tag("food.png")%><br>Food</li>
+      <li><%= image_tag("hygiene.png")%><br>Hygiene Products</li>
+      <li><%= image_tag("clothing.png")%><br>Clothing</li>
+      <li><%= image_tag("furniture.png")%><br>Furniture</li>
+      <li><%= image_tag("toys.png")%><br>Toys</li>
+      <li><%= image_tag("books.png")%><br>Books</li>
+      <li><%= image_tag("baby.png")%><br>Baby Items</li>
+      <li><%= image_tag("ect.png")%><br>And More</li>
     </ul>
-    <p>
-      Recycling and redistributing new and used things to people in need such as clothing, furniture, hygiene products, toys, books, house hold items, and food.
-    </p>
+
+    <div class="row">
+      <div class="large-10 large-centered columns">
+        <p>We also welcome leads, and information to other organizations that can provide assistance. Recycling and redistributing new and used things to people in need such as clothing, furniture, hygiene products, toys, books, house hold items, and food.</p>
+      </div>
+    </div>
   </div>
-  <!-- Needs a new source! -->
-  <!-- <img src="" class="one-third-image"> -->
-</div>
 
+  <div class="faceook-box">
+    <h3>Follow us on Facebook!</h3>
+    <div class="fb-page" data-href="https://www.facebook.com/pages/Humble-Hearts-Organization/477172595731068?" data-width="500" data-height="300" data-hide-cover="false" data-show-facepile="true" data-show-posts="true"><div class="fb-xfbml-parse-ignore"><blockquote cite="https://www.facebook.com/pages/Humble-Hearts-Organization/477172595731068"><a href="https://www.facebook.com/pages/Humble-Hearts-Organization/477172595731068">Humble Hearts Organization</a></blockquote></div></div>
```

Comment:
This facebook box isn't showing and I can't figure out why. I'm using the code as facebook tells me to use it. I remember it was working during the hackathon.